R-packages/evalcast/R/get_predictions.R

```diff
@@ -54,7 +54,7 @@ get_predictions <- function(forecaster,
                             geo_type,
                             geo_values = "*",
                             apply_corrections = NULL,
-                            signal_aggregation = c("list", "wide", "long"),
+                            signal_aggregation = "list",
```

Comment:
I think you should revert this change. My understanding is that the old version was the right way to do it. The `match.arg()` call below pulls out the first matching argument. The advantage of the old way is that the default shows the available options (rather than just the first).

Reply:
For whatever reason I thought this was causing a bunch of issues but apparently it is ok.  Reverted.